### PR TITLE
Feature/224 load let after execute

### LINKED ISSUE #224: Display LET distribution (2D) once plan execution is completed

### PR DESCRIPTION
**Pull request by ljelen**
closes #224 

Please, check if this Let Container class is appropriate. I want to add something similar to model.dos and model.plans

### CODE REVIEW

File: pytripgui/model/let_container.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        # update model
+        self.lets.append(let)
+        return let
+
+    def import_let_from_list(self, let_list):
+        for let in let_list:
+            self.lets.append(let)
+
+        return let_list[-1]
```
Comment (grzanka) [inline pytripgui/model/let_container.py]: Why only last one is loaded ?

**Comment by grzanka** [comment]
It looks OK at first glance.

File: pytripgui/model/main_model.py [hunk 2]
```diff
@@ -23,7 +22,7 @@ def __init__(self, app):
         self.vdx = None  # Only one VDX is allowed
         self.plans = []
         self.dos = []   # TODO: change to self.doss (plural s)
-        self.let = []   # TODO: change to self.lets (plural s)
+        self.lets = LetContainer()
```
Comment (nbassler) [inline pytripgui/model/main_model.py]: could also be 
let_list = 
when plural s doesn't work well. (in particular for dos, where plural s would be ambigous.)

File: pytripgui/model/let_container.py (hunk 1 above)
Comment (ljelen) [reply pytripgui/model/let_container.py]: i loads every one, but return last one

File: pytripgui/model/main_model.py (hunk 2 above)
Comment (ljelen) [reply pytripgui/model/main_model.py]: right :D

**Comment by grzanka** [comment]
@nbassler what do you think of display overlays when both dose and LET cubes are calculated (there may be even more cubes: bio dose, alpha, rbe,...).
I think we should have then two (or more) tabs, all with CT data displayed.
When plan calculation is completed, in all tabs relevant overlays (dose, LET etc) should be displayed.

Other solution is to have a single tab with CT and user should choose one from many overlays to put on top of it.

**Comment by grzanka** [comment]
After merging dose display, rebase this one onto newest master, then add also "multiple tab" solution.

**Pull request merged**